src/sentry/mediators/external_requests/util.py

```diff
@@ -35,3 +40,29 @@ def validate(instance, schema_type):
         return False
 
     return True
+
+
+def send_and_save_request(sentry_app, org_id, event, **kwargs):
```

Comment:
@nolachen Why is `url` an optional parameter?

Reply:
@scefali Fixed, it should not be